pkg/provision/mbus/network.go

```diff
@@ -132,7 +132,7 @@ func (n *Network) listPublicIps() (interface{}, mw.Response) {
 			if err != nil {
 				return nil, mw.Error(errors.Wrap(err, "failed to load deployment"))
 			}
-			workloads := deployment.ByType(zos.PublicIPType)
+			workloads := deployment.ByType(zos.PublicIPv4Type)
```

Comment:
Should PublicIPType type be added as well?

Reply:
Yes, will add it as well.